### PR TITLE
PHPDocs Application -> HttpKernelInterface

### PR DESCRIPTION
**Pull request by SpacePossum**
replacement of https://github.com/silexphp/Silex/pull/1193
(could not reopen that one because of a forced push)

### CODE REVIEW

File: src/Silex/ServiceProviderInterface.php [hunk 1]
```diff
@@ -23,6 +23,8 @@
      *
      * This method should only be used to configure services and parameters.
      * It should not get services.
+     *
+     * @param Application $app
```
Comment (fabpot) [inline src/Silex/ServiceProviderInterface.php]: Can be removed as all the info are already part of the function definition.

File: src/Silex/ServiceProviderInterface.php [hunk 2]
```diff
@@ -32,6 +34,8 @@ public function register(Application $app);
      * This method is called after all services are registered
      * and should be used for "dynamic" configuration (whenever
      * a service must be requested).
+     *
+     * @param Application $app
```
Comment (fabpot) [inline src/Silex/ServiceProviderInterface.php]: same here

Comment (SpacePossum) [inline src/Silex/ServiceProviderInterface.php]: check, thanks and done :)

**Comment by fabpot** [comment]
Thank you @SpacePossum.

**Pull request merged**